apps/atomy-q/API/database/factories/VendorInvitationFactory.php

```
@@ -0,0 +1,75 @@
+
+    public function declined(): static
+    {
+        return $this->state(fn (array $attributes) => [
+            'status' => 'declined',
+            'responded_at' => now(),
+        ]);
+    }
+
+    public function forRfq(Rfq $rfq): static
+    {
+        return $this->state(fn (array $attributes) => [
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)